### ISSUE TITLE
What elements of an accessibility scan can be affected by users/ theme/ framework?

### ISSUE THREAD

**Issue by mvogelgesang** [GSA/EDX#602]
## Summary

When measuring accessibility within an agency, there are different layers at which an accessibility problem may be addressed. For instance:

🟢 - High impact
🟡 - Possible impact
🔴 - No impact

| Accessibility Criteria | Website Framework | Website Theme | Content Creator | Developer |
|---|---|---|---|---|
|Image alt text | 🟡 | 🔴 | 🟢 | 🟡 |
|Color Contrast | 🔴 | 🟢  | 🔴  | 🟡 |
|<meta> - Viewport | 🟡 | 🟢   | 🔴  | 🟡 |
|Content width | 🟡 | 🟡 | 🔴 | 🟡 |
|Bypass (skip to content)| 🔴 | 🟢 | 🔴  |🟡 |
|<title> (present)| 🟡  | 🟢 | 🔴  |🟡 |
|<title> (quality)| 🔴   | 🔴  | 🟢| 🟡 |
|<html> (lang)| 🟡 | 🟡 | 🟡 | 🟡 |

## Additional context and links

W3C has an [Accessibility Roles and Responsibilities Mapping framework](https://www.w3.org/WAI/EO/wiki/ARRM_Project_-_Accessibility_Roles_and_Responsibilities_Mapping) which may provide some insights into this. hat tip to @mgifford for the link via USWDS Slack

## Checklist

List below the specific actions to be taken

- [ ] item 1
- [ ] item 2

**Comment by AKM399** [comment]
@hatfieldjm4 is this useful guidance for website evaluations?

**Comment by hatfieldjm4** [comment]
I don't think this would add much value to the website evals because I don't know how actionable they would perceive it. We could link to something like this for awareness of the website responsibility areas versus accessibility impacts. I'm interested in Mark's take.

**Comment by mgifford** [comment]
I like how axe deals with this https://github.com/dequelabs/axe-core/blob/develop/doc/issue_impact.md

As a Drupal Core Accessibility Maintainer, I'm also confident that there is a lot that the website framework can impact above and beyond the theme. 

Having a framework that is built for accessibility (like Drupal), and a theme (like the USWDS) that supports best practices is great. 

Frameworks can also impact accessibility of the content. ATAG 2.0 AA is a great resource for this. I represented Drupal as part of this European Commission funded project - https://accessibilitycluster.com/main-results

**Comment by mgifford** [comment]
Happy to help @mvogelgesang for the Angular/React sites, I'd recommend looking at the work of one of my colleagues [https://accessibility.civicactions.com/posts/automated-accessibility-testing-leveraging-github-actions-and-pa11y-ci-with-axe](https://accessibility.civicactions.com/posts/automated-accessibility-testing-leveraging-github-actions-and-pa11y-ci-with-axe) and more recently [https://opensource.com/article/23/2/automated-accessibility-testing](https://opensource.com/article/23/2/automated-accessibility-testing)  
  
Also, although I'd love to see CKEditor adopt more of ATAG, worth checking out [https://ckeditor.com/github-writer](https://ckeditor.com/github-writer/) but it is pretty hard to mess up the semantics of Markdown.  
  
Perhaps an easier tool to incorporate would be [https://sa11y.netlify.app](https://sa11y.netlify.app) or even just an amazing tool like [https://accessibilityinsights.io/](https://accessibilityinsights.io/)  
  
We collect a lot of these resources and practices here [https://accessibility.civicactions.com/](https://accessibility.civicactions.com/)

**Comment by anatecture** [comment]
Done by chrome extension
